### PR TITLE
 Add gettext.js@0.5.3 w/ git auto-update 

### PR DESCRIPTION
**Pull request by abdulamite**
Pull request for issue: #11902 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/guillaumepotier/gettext.js.git
 * Official website (optional, not the repository): http://guillaumepotier.github.io/gettext.js/
 * NPM package url (optional): [MIT](https://github.com/dobtco/formbuilder/blob/gh-pages/LICENSE.md)
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 2
   - Count of stars: 16
   - Count of forks: 9

# Essential checklist
  * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by abdulamite** [comment]
so I already see that the commit messages are all over the place and need to be squashed I would love to be able to get some help resolving this

**Comment by abdulamite** [comment]
@PeterDaveHello do you think I could get someone to review this PR

**Comment by abdulamite** [comment]
@extend1994 Thank you!!

File: ajax/libs/gettext.js/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "gettext.js",
+  "filename": "gettext.min.js",
+  "version": "0.5.3",
+  "description": "gettext.js is a lightweight (3k minified!) yet complete and accurate GNU gettext port for node and the browser. Manage your i18n translations the right way in your javascript projects.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/guillaumepotier/gettext.js.git"
+  },
+  "homepage": "http://guillaumepotier.github.io/gettext.js/",
```
Comment (maruilian11) [inline ajax/libs/gettext.js/package.json]: Please use `https`, thanks.

**Comment by abdulamite** [comment]
so I updated the file, but I am having a hard time pushing the changes to my pull request. so I updated my file and I ran the following: 
```git push https://github.com/abdulamite/cdnjs.git gettext -f ```

**Comment by PeterDaveHello** [comment]
@abdulamite 👍

**Comment by abdulamite** [comment]
Thank you everyone!

**Pull request merged**